pywren/wren.py

```diff
@@ -60,7 +59,7 @@ def remote_executor(config=None, job_max_runtime=3600):
 
     AWS_REGION = config['account']['aws_region']
     SQS_QUEUE = config['standalone']['sqs_queue_name']
-    invoker = invokers.SQSInvoker(AWS_REGION, SQS_QUEUE)
+    invoker = queues.SQSInvoker(AWS_REGION, SQS_QUEUE)
 
```

Comment:
I think the invokers should be a module and the queues.py just provide some implementations. So it's nice to access all invokers from invokers.py. 
Personally I feel having a separate`queues.py` file is not very necessary for now, but we might expand more in the future.